src/pages/index.js

```diff
@@ -1,190 +1,217 @@
-import React from 'react';
-
 import {
   Box,
+  ButtonGroup,
   Flex,
   Heading,
   Text,
   useTheme,
-  ButtonGroup,
 } from '@chakra-ui/core';
-import Layout from '../components/Layout';
-import ContentBlock from '../components/ContentBlock';
+import { graphql, StaticQuery } from 'gatsby';
+import React from 'react';
 import Button from '../components/Button';
+import ContentBlock from '../components/ContentBlock';
+import ErrorBoundary from '../components/ErrorBoundary';
+import Layout from '../components/Layout';
 
 export default () => {
   const theme = useTheme();
   return (
-    <Layout>
-      <Flex direction="column" align="center" justify="center">
-        <ContentBlock
-          layout="left"
-          imageSource="assets/home-header-bg" // @TODO :: Pass this to cloudinary
-          backgroundColor="#000"
-          backgroundMode="fade"
-        >
-          <Box pt={16} pb={[72, 72, 145]} color="white">
-            <Heading mb={theme.spacing.lg}>
-              <Text
-                lineHeight="0.5"
-                fontWeight="900"
-                fontSize={['50px', '50px', '50px', '72px']}
-                fontFamily={theme.fonts['heading-slab']}
-                textTransform="uppercase"
-              >
-                Rebuild
-              </Text>
-              <Text
-                fontSize={['100px', '100px', '100px', '150px']}
-                fontWeight="900"
-                lineHeight="1"
-                fontFamily={theme.fonts['heading-slab']}
-                textTransform="uppercase"
+    <ErrorBoundary>
+      <StaticQuery
```

Comment:
Same query question here, can we wrap only the dynamic bits of markup in this query?

Reply:
should be fixed now.